addons/stock/views/stock_move_line_views.xml

```diff
@@ -30,6 +30,17 @@
             </tree>
         </field>
     </record>
+
+    <record id="action_reverse_inventory_adjustment" model="ir.actions.server">
+        <field name="name">Reverse Inventory Adjustment</field>
+        <field name="model_id" ref="stock.model_stock_move_line"/>
+        <field name="binding_model_id" ref="stock.model_stock_move_line"/>
+        <field name="state">code</field>
+        <field name="code">
+            action = records.action_revert_inventory()
+        </field>
+    </record>
+
```

Comment:
Can you move this down after `<record id="view_stock_move_line_kanban" model="ir.ui.view">` actions and above `record id="stock_move_line_action" model="ir.actions.act_window">`? It will keep the file cleaner if we keep the `ir.ui.view` records together in this case.